HISTORY.rst

```diff
@@ -27,6 +27,7 @@
 
 **Bug fixes**
 
+- 1093_: [SunOS] memory_maps() shows wrong 64 bit addresses
```

Comment:
I didn't see the mismatch with other numbers (because usually they are quite low, and there is no chances to get signed integer overflow), but semantic should be the same - these numbers can not be negative.